src/lib/collection/message-handling.collection.ts

```diff
@@ -67,7 +67,9 @@ export class MessageHandlingCollection {
    */
   getMessage(handler: any): any {
     const messages = this._collection
-      .filter((pair: MessageHandlerPair) => pair.handler === handler)
+      .filter((pair: MessageHandlerPair) =>
+        pair.handlers.some((value: any) => value === handler)
+      )
```

Comment:
Why did you introduce a breaking change?
Can you please create a new collection by preserving this one to be used with a single handler? Otherwise, any project that uses this lib should change its configuration.